portal-web/test/functional/com/liferay/portalweb/macros/site/SiteTemplates.macro

```diff
@@ -442,10 +442,16 @@ definition {
 		var groupId = JSONGroupAPI._getGroupIdByName(
 			groupName = ${groupName},
 			site = "true");
+		var databaseType = PropsUtil.get("database.type");
 
-		var mysqlStatement = "SELECT modifiedDate FROM Layout WHERE name LIKE '${pageNameFilter}' AND groupId = '${groupId}' AND hidden_ = 0;";
+		if (${databaseType} == "mysql") {
```

Comment:
@magjed4289 can you test if this query works for both without the if statement?

```
var sqlStatement = "SELECT modifiedDate FROM Layout WHERE name LIKE '${pageNameFilter}' AND groupId = '${groupId}' AND hidden_ = 0;";
```

Reply:
from what I remember the Layout is not recogniseable in postgres, only the "layout" works.